examples/skr/aci/README.md

```diff
@@ -16,16 +16,109 @@ The MAA endpoint shall be the same as the one specified in the SKR policy during
+
+There are two options for generating security policies:
+
+1. The ARM template can be used directly to generate a security policy.
+2. A config file can be used to generate a policy fragment for a particular image.
+This is useful in cases where stable key release policies are required.
+This option is called "image-attached fragments".
+
+#### Option 1
 
-The ARM template can be used directly to generate a security policy.
 The following command generates a security policy and automatically injects it into the template.
-Make sure `--debug-mode` option is included so that the generated policy allows shelling into container to see the released key in this example.
+Make sure `--debug-mode` option is included so that the generated policy allows shelling into container to see the released key in this example. Note that `--debug-mode` is only used as an example and not recommended for production.
 
 ```shell
 az confcom acipolicygen -a aci-skr-arm-template.json --debug-mode
 ```
 
+#### Option 2
+
+The following command generates a security policy and attaches it to the container in an OCI-compliant registry like Azure Container Registry (ACR). To do this, a copy of the SKR container must be made and pushed to an ACR instance with write access. This can be done with:
+
+```bash
+az acr login -n <my-registry>
+docker pull mcr.microsoft.com/aci/skr:2.9
+docker tag mcr.microsoft.com/aci/skr:2.9 <my-registry>.azurecr.io/skr:2.9
+docker push <my-registry>.azurecr.io/skr:2.9
+```
+
+Create a configuration file with the container details as illustrated below:
+
+```json
+{
+    "containers": [
+          {
+            "name": "skr-sidecar-container",
+            "properties": {
+              "command": [
+                "/skr.sh"
+              ],
+              "environmentVariables": [
+                {
+                  "name": "LogFile",
+                  "value": "<optional-logfile-path>"
+                },
+                {
+                  "name": "LogLevel",
+                  "value": "<optional-loglevel-trace-debug-info-warning-error-fatal-panic>"
+                }
+              ],
+              "image": "<my-registry>/skr:2.9"
+            }
+        }
+    ]
+}
+```
+
+Save this file to the current directory as `fragment_config.json`. Then run the command to generate a policy fragment and upload it to the image registry:
+
+```bash
+az confcom acifragmentgen -i fragment_config.json --debug-mode --upload-fragment --image-target <my-registry>/skr:2.9 --key <path-to-my-key> --chain <path-to-my-cert-chain> --svn 1 --namespace <my-namespace>
+```
+
+To verify that a policy fragment is coming from a valid source, it must be COSE signed via a key and certificate chain.
+Instructions and guidelines on how to do that for a development deployment are given [here](https://github.com/Azure/azure-cli-extensions/tree/main/src/confcom/samples/certs/README.md) but other methods to generate a cert chain and key may be used.
```

Comment:
I would put this above the confcom command so that the user knows to do this first